src/game/PetAI.cpp

```diff
@@ -335,6 +335,16 @@ void PetAI::UpdateAI(const uint32 diff)
             return;
         }
 
+        if (!m_creature->HasInArc(2 * M_PI_F / 3, victim))
+        {
+	        m_creature->SetInFront(victim);
```

Comment:
It looks like you may have hit the tab twice and thought that you could delete the whole tab by pressing backspace but in fact you only deleted a single whitespace? That would explain the odd number of 7, and the 'double tab' you do in line 342 and 345 ?